src/components/CharacterLimitMainText/components/CharacterLimit.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import { render } from '@testing-library/react';
+
+import CharacterLimit from './CharacterLimit';
+
+describe('CharacterLimit Component', () => {
+  it('should apply the "exceeded" class when character limit is exceeded', () => {
```

Comment:
Esse bloco de it tá muito grande, fora do estilos que escrevemos.



```suggestion
  describe(' when character limit is exceeded', () => {
    it('apply the "exceeded" class', () => {
```

Reply:
done